### DISCUSSION TITLE
Next Output File Tracing -  next.config.js option 'outputFileTracingIncludes' not working - appInsights

### DISCUSSION THREAD

**Discussion by chrislicodes** [vercel/next.js#47293] [Help]
### Summary

Hello,

I want to avoid using a custom server and load appInsights before server startup.

My next.config.js looks like this:

```js
/** @type {import('next').NextConfig} */
// @ts-check
module.exports = {
  output: 'standalone',
  reactStrictMode: true,
  swcMinify: true,
  experimental: {
    outputFileTracingIncludes: {
      '/': ['./load-appInsights.js'],
    },
  },
};
```

In my root I have a file called `load-appInsights.js` that starts with

```js
const appInsights = require('applicationinsights');
```

I want to pre-load this file before running the standalone server.js like 

```bash
node --require ./load-appinsights.js server.js,
```

For that the standalone folder needs to include `load-appInsights.js` and all necessary dependencies. Is there a way to solve this?
The setting for `outputFileTracingIncludes` doesn't seem to work.

**Comment by needleshaped** [comment]
Trying to work similar configuration out with https://github.com/expatfile/next-runtime-env: 
- `CMD node -e "const { configureRuntimeEnv } = require('next-runtime-env/build/configure'); configureRuntimeEnv()"  && node server.js`
- and
```
  output: 'standalone',
  experimental: {
    outputFileTracingIncludes: {
      '/': ['./next-runtime-env/**/*'],
    },
  },
```

However, also without a success

**Comment by Jokinen** [comment]
A similar discussion: https://github.com/vercel/next.js/discussions/38004

Does not contain a solution at the time of posting.

**Reply by Jokinen** [reply]
Another similar discussion that presents one workaround: https://github.com/vercel/next.js/discussions/35792
